src/strategies/adventureland/examples.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "adventureland"
+    },
+    "network": "56",
+    "addresses": [
+      "0xa478c2975ab1ea89e8196811f51a7b7ade33eb11",
+      "0xeF8305E140ac520225DAf050e2f71d5fBcC543e7",
+      "0x1AeE7eBabdAa9afe11B2abD2748a548316E6c065"
+    ],
+    "snapshot": 7120529
+  },
+  {
+    "name": "Synthetix council",
```

Comment:
Hi @AdventurelandDeFi can u plz remove this second if it is not required, Thanks 


LGTM @bonustrack

Reply:
done